meta-facebook/aalc-rpu/src/platform/plat_hwmon.c

```diff
@@ -415,7 +418,13 @@ uint8_t pwm_control(uint8_t group, uint8_t duty)
 	static int64_t auto_tune_time;
 	if (!get_status_flag(STATUS_FLAG_AUTO_TUNE)) {
 		auto_tune_time = k_uptime_get();
-		set_pwm_group(group, 0);
+		return 0;
+	}
+
+	// keep 0 duty when first enabling auto tune
```

Comment:
could you please expand on the comment to describe why it's required that we return early on first call?

Reply:
The current design is that when the AC is on, the pump will remain at duty 0 until someone turns on auto tune.